server/src/main/java/org/opensearch/action/support/clustermanager/ClusterManagerNodeReadRequest.java

```diff
@@ -47,6 +47,8 @@ public abstract class ClusterManagerNodeReadRequest<Request extends ClusterManag
 
     protected boolean local = false;
 
+    protected boolean shouldCancelOnTimeout = false;
+
     protected ClusterManagerNodeReadRequest() {}
 
     protected ClusterManagerNodeReadRequest(StreamInput in) throws IOException {
```

Comment:
changes for serialization/ deserialization to handle shouldCancelOnTimeout flag based on version in constructor and writeTo method.